app/views/collection/_contributors_body.html.slim

```diff
@@ -49,7 +50,7 @@ h3 =t('.all_collaborator_emails')
 p=link_to "Find recommendations for project newsletters.", "https://content.fromthepage.com/crowdsourcing-project-newsletter/"
 p#collaborators
   -unless @all_collaborators.empty?
-    =@all_collaborators + ", support@fromthepage.com"
+    =format_collaborators
```

Comment:
This is the only place that needs support@fromthepage.com.

Reply:
@saracarl right, format_collaborators won't affect the CSV output, it's only there to remove a comma from the view when the string already includes one.